cmd/virtual-kubelet/commands/root/flag.go

```diff
@@ -78,6 +78,8 @@ func installFlags(flags *pflag.FlagSet, c *Opts) {
 	flags.StringVar(&c.TraceSampleRate, "trace-sample-rate", c.TraceSampleRate, "set probability of tracing samples")
 
 	flags.DurationVar(&c.InformerResyncPeriod, "full-resync-period", c.InformerResyncPeriod, "how often to perform a full resync of pods between kubernetes and the provider")
+	flags.DurationVar(&c.StartupTimeout, "startup-timeout", c.StartupTimeout, "How long to wait for the virtual-kubelet to start")
+	flags.BoolVar(&c.WaitForStartup, "wait-for-startup", c.WaitForStartup, "Wait for pod controller to start, before starting up the node")
```

Comment:
I'm not sure we need this. If the startup times out we should just exit.

Reply:
I don't know what the project's perspective is on breaking behaviour. If we do something like waiting on a channel, or a Ready() method, then the node wont come up until the podcontroller is started. 

If you think it's okay changing behaviour, I'm all for making things simpler.